README.md

```diff
@@ -2,7 +2,135 @@
 
 # SNVPhyl nf-core Pipeline
 
-This is the in-development nf-core-based pipeline for SNVPhyl.
+This is the [nf-core](https://nf-co.re/)-based pipeline for [SNVPhyl](https://snvphyl.readthedocs.io/en/). The SNVPhyl (Single Nucleotide Variant PHYLogenomics) pipeline identifies Single Nucleotide Variants (SNV) within a collection of microbial genomes and constructs a phylogenetic tree from those SNVs. This pipeline is designed to be integrated into [IRIDA Next](https://github.com/phac-nml/irida-next). However, it may be run as a stand-alone pipeline.
+
+# Input
+
+Input is provided to SNVPhyl in the form of a samplesheet (passed as `--input samplesheet.csv`). This samplesheet is a CSV-formated file, which may be provided as a URI (ex: a file path or web address), and has the following format:
+
+| sample  | fastq_1                    | fastq_2                    | assembly                     |
+| ------- | -------------------------- | -------------------------- | ---------------------------- |
+| SAMPLE1 | /path/to/sample1_fastq1.fq | /path/to/sample1_fastq2.fq | /path/to/sample1_assembly.fa |
+| SAMPLE2 | /path/to/sample2_fastq1.fq |                            |                              |
+
+The columns are defined as follows:
+
+- `sample`: The unique sample identifier to associate with the reads (and optionally the assembly).
+- `fastq_1`: A URI (ex: a file path or web address) to either single-end FASTQ-formatted reads or one pair of pair-end FASTQ-formatted reads.
```

Comment:
Is single-end analysis supported right now? If not, I would leave out mentioning single-end analysis.

Reply:
Yup, I added single-end a few PRs back. I can't remember which one, but it works.